assets/ajax-cart.js.liquid

```diff
@@ -0,0 +1,627 @@
+/*============================================================================
+  (c) Copyright 2015 Shopify Inc. Author: Carson Shold (@cshold). All Rights Reserved.
+
+  Plugin Documentation - http://shopify.github.io/Timber/#ajax-cart
+
+  Ajax the add to cart experience by revealing it in a side drawer
+
+  This file includes:
+    - Basic Shopify Ajax API calls
+    - Ajax cart plugin
+
+  This requires:
+    - jQuery 1.8+
+    - handlebars.min.js (for cart template)
+    - modernizer.min.js
+    - snippet/ajax-cart-template.liquid
+
+  JQUERY API (c) Copyright 2009-2015 Shopify Inc. Author: Caroline Schnapp. All Rights Reserved.
+  Includes slight modifications to addItemFromForm.
+==============================================================================*/
+if ((typeof Shopify) === 'undefined') { Shopify = {}; }
+
+/*============================================================================
+  Basic JS Helper Functions
+==============================================================================*/
+function urlParams (name) {
+  name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
+  var regex = new RegExp("[\\?&]" + name + "=([^&#]*)"),
+      results = regex.exec(location.search);
+  return results == null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
+}
+
+/*============================================================================
+  API Helper Functions
```

Comment:
One reason is to reduce the need for another HTTP request for the globally hosted api file. The other is for adding functionality to `Shopify.addItemFromForm`, mainly allowing it to use a form element (`evt.target` from a submitted form) rather than just an ID, and adding a custom callback if defined.